common/components/Footer/index.tsx

```diff
@@ -94,7 +94,7 @@ const LINKS_SOCIAL = [
 
 interface Props {
   latestBlock: string;
-};
+}
```

Comment:
Looks like this file got formatted without semis

Reply:
@HenryNguyen5 Is that supposed to have semis? I have semis set to true, but it's not adding the semi there. The semi is added on line 96. confusing.